client/src/pages/CreateWorkout/CreateWorkout.jsx

```diff
@@ -68,17 +78,17 @@ const CreateWorkout = () => {
       <div className=" mt-4 flex flex-wrap justify-center sm:justify-between md:mr-11 ">
         <div className="w-3/4">
           <h2 className=" my-0 truncate font-bold tracking-wide">
-            {newWorkout ? "Add Workout" : videoInfo.title}
+            {workoutId ? videoInfo.title : "Add Workout"}
```

Comment:
nice - this is a much simpler check over keeping  `newWorkout`  as a separate state :)